### DISCUSSION TITLE
External map design not working?

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#520] [Q&A] [answered]
@devemux86 

Yesterday I "played" a little bit with an extracted xml file from the crusier apk and the assets/vtm folder.
It looks like these are the files for the map designs which are available in cruiser.
Well, I opened one and did some changings and saved it under another name.
(for a first test I just changed the hex color code of the motorway road)

After that I tried to load it as an external map design as *.xml file but...nothing happend, the map design did not change.

Do you have an idea why?

**Comment by devemux86** [comment, marked as answer]
> I tried to load it as an external map design as *.xml file

External map themes are loaded as zip files (there is a message in the screen).
You can skip the extra resources like icons, etc., if use the same as Cruiser.

For example see how @MotoUKRider provides the Motorider theme:
- https://github.com/devemux86/cruiser/discussions/176#discussioncomment-8270684

---

By the way the map themes are open source and are maintained here:
https://github.com/mapsforge/vtm/tree/master/vtm-themes/resources/assets/vtm
